optuna_dashboard/ts/components/AppDrawer.tsx

```diff
@@ -300,6 +303,19 @@ export const AppDrawer: FC<{
                 <ListItemText primary="Note" sx={styleListItemText} />
               </ListItemButton>
             </ListItem>
+            <ListItem key="Settings" disablePadding sx={styleListItem}>
+              <ListItemButton
+                component={Link}
+                to={`${URL_PREFIX}/studies/${studyId}/settings`}
```

Comment:
How about using pop-up to show setting page? I roughly created my example [here](https://github.com/gen740/optuna-dashboard/compare/add_color_scale_setting...keisuke-umezawa:optuna-dashboard:example-popup), although it is very early implementation and needs improvement. 

Some reasons:
1. You can use ${URL_PREFIX}/settings instead of "${URL_PREFIX}/studies/${studyId}/settings" here.
2. You can show change during tweaking theme on settings.

Reply:
Also, the place of setting should be below because it is for setting for all pages, imo.
<img width="65" alt="スクリーンショット 2024-01-01 11 49 14" src="https://github.com/optuna/optuna-dashboard/assets/1277089/89a29db5-7a70-4531-a38f-28149ee05bed">

You can see this example [here](https://github.com/gen740/optuna-dashboard/compare/add_color_scale_setting...keisuke-umezawa:optuna-dashboard:example-popup)